packages/core/lib/blockUtils.js

```diff
@@ -1,5 +1,6 @@
 const _ = require('lodash')
 const webpackMerge = require('webpack-merge')
+const webpackBlocksMerge = webpackMerge.smartStrategy({ 'resolve.extensions': 'prepend' })
```

Comment:
For now I left it inlined, but in case there will be more custom behavior I'd recommend moving this object to something like `merge-strategies.js`